stdlib/Dates/src/deprecated.jl

```diff
@@ -65,3 +65,6 @@ for op in (:+, :-)
         end
     end
 end
+
+@deprecate argerror(msg::String) ArgumentError(msg)
+@deprecate argerror() nothing
```

Comment:
I think you need:
```suggestion
@deprecate argerror(msg::String) ArgumentError(msg) false
@deprecate argerror() nothing false
```
otherwise it will be exported.

Reply:
Yes, thank you.